content/en/blog/_posts/2023-11-02-mid-cycle-1.29.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: blog
+title: 'Kubernetes Removals, Deprecations, and Major Changes in Kubernetes 1.29'
+date: 2023-11-09
```

Comment:
On the SIG Docs call, the proposed publication date is Nov 2. 
```suggestion
date: 2023-11-02
```

Reply:
At the 11-01-23 Release Team meeting, we discussed moving the publication date until the week after Kubecon.

Reply:
@kcmartin @a-mccarthy changed to Nov 16th after Kubecon